cfg/admin.txt

```diff
@@ -1,2 +1,23 @@
-jmwturner role=admin
-deitylink role=admin
+unid role=admin
+abi79 role=admin
+arcalane role=admin
+asanadas role=admin
+bobbehluvspropane role=admin
```

Comment:
> Also it is in .gitignore my bad, people just need to update their .gitignores since pulling from the repo doesn't work since .gitignore ignores itself lmao.

That's only if you gitignore the gitignore, which is already your own fault.